etc/xivo/asterisk/xivo_globals.conf

```diff
@@ -24,6 +24,7 @@ XIVO_FWD_USER_BUSY_ISDA = 0
 XIVO_FWD_USER_CONGESTION_ISDA = 0
 XIVO_FWD_USER_CHANUNAVAIL_ISDA = 0
 XIVO_FWD_SCHEDULE_OUT_ISDA = 0
+XIVO_FWD_USER_DND_ISDA = 1
```

Comment:
This prevents the "find_user" forwarding message routine from being run twice when forwarding to a DND user.